omod/src/main/webapp/fragments/systemadmin/accounts/personDetails.gsp

```diff
@@ -38,6 +42,19 @@
                     <th valign="top">${ui.message("Person.gender")}</th>
                     <td valign="top">{{genders[person.gender]}}</td>
                 </tr>
+
+                <% customPersonViewFragments.each { sect ->
+                    if(sect.extensionParams.type == 'personAttribute') { %>
+                    ${ ui.includeFragment(sect.extensionParams.provider, sect.extensionParams.fragment, [
+                            label : ui.message(sect.extensionParams.label),
+                            personId : account.person.personId,
+                            personUuid : account.person.uuid,
+                            initialValue : ui.message(account.getPersonAttribute(sect.extensionParams.uuid) != null ?
+                                    account.getPersonAttribute(sect.extensionParams.uuid).value : '')
```

Comment:
Can you indent the above correctly?

Reply:
That is the continue part of the above line. Does it contain wrong indent?